pybedlite/bed_record.py

```diff
@@ -188,3 +196,27 @@ def as_bed_line(self, number_of_output_fields: Optional[int] = None) -> str:
         )
         fields = self.bed_fields[:number_of_output_fields]
         return "\t".join(fields)
+
+    @classmethod
+    def from_interval(cls: Type["BedRecord"], interval: Interval) -> "BedRecord":
```

Comment:
I fixed the circular import (see above, the only conflict in `bed_record.py` was with the type hint in this signature)

Reply:
Can you try:
```
    def from_interval(cls: Type["BedRecord"], interval: "Interval") -> "BedRecord":
```
and if that doesn't work, lgtm?